### ISSUE TITLE
Add load average information to resources API

### ISSUE THREAD

**Issue by stgraber** [lxc/incus#828]
Currently the resources API (`/1.0/resources`) does not provide system load information.
While system load average isn't exactly the best metric ever, it would be useful in some situations like our automated re-balancing.

To address that, we should add a new `load` section to the resources API which would contain:
 - Average1Min
 - Average5Min
 - Average10Min
 - Processes

The first 3 refer to the first 3 fields from /proc/loadavg, the last one would be a total process count on the system which one can do through ReadDir of /proc and counting the number of numeric-only directories.

`incus info --resources` should also be extended as part of this to show the extra fields.

**Comment by camnwalter** [comment]
Hi! Can you assign this one to me?
It looks like I just need to add a `Load` field to the `api.Resources` struct, via a new GetLoad function. I can copy the implementation of `getLoadAvgs` in the cluster, and for the process count I just can just read /proc for directories with names as only numbers.

**Comment by stgraber** [comment]
Yep and also implement the part that fills the total process count.
